src/arcade/eng/common/core-templates/post-build/post-build.yml

```diff
@@ -159,7 +159,7 @@ stages:
 
     - job:
       displayName: Signing Validation
-      condition: and( eq( ${{ parameters.enableSigningValidation }}, 'true'), ne( variables['PostBuildSign'], 'true'))
+      condition: and(succeeded(),  eq( ${{ parameters.enableSigningValidation }}, 'true'), ne( variables['PostBuildSign'], 'true'))
```

Comment:
this doesn't look correct since it'd make this job fail if the previous one (`NuGet Validation`) failed but these jobs are independent

Reply:
that said, I don't know why the `NuGet Validation` job uses `succeededOrFailed` since it's the first job in the stage so this should be a no-op

Reply:
ah I guess this catches the case where the build is canceled: https://learn.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops&preserve-view=true#succeededorfailed

> You may want to use `not(canceled())` instead when there are previous skipped jobs in the dependency graph.

Reply:
Maybe for the `job` conditions we can just leave them alone.  It's less obvious what their behavior should be.